src/main/resources/io/anserini/qa/english-stoplist.txt

```diff
@@ -0,0 +1,174 @@
+a
```

Comment:
I used one of those available online. I think it's from here: http://www.ranks.nl/stopwords